src/types/program.types.ts

```diff
@@ -14,19 +26,6 @@ export enum DealStatus {
 	PENDING,
 }
 
-export type Deal = {
-	borrower: PublicKey;
-	principal: BN;
-	financingFeePercentage: number;
-	principalAmountRepaid: BN;
-	interestAmountRepaid: BN;
-	timeToMaturityDays: number;
-	goLiveAt: BN;
-	createdAt: BN;
-	leverageRatio: number;
-	underwriterPerformanceFeePercentage: number;
-};
-
 export type PrincipalRepaymentType = { principal: {} };
```

Comment:
Is it possible to get the type for these also from `CredixTypes`

Reply:
I have looked into it it doesn't seem to be so. I may be wrong though but I didn't find a way in the source code (no documentation..)